deploy/helm/magda-dev.yml

```diff
@@ -131,7 +131,6 @@ correspondence-api:
     requests:
       cpu: 100m
       memory: 0
-  externalUrl: "https://magda-dev.terria.io"
```

Comment:
Will this cause any trouble for dev site deployment (master branch)?
It seems `externalUrl` now is from `externalUrl=http://$CI_COMMIT_REF_SLUG.magda-dev.terria.io`
Does that mean dev site won't get value "https://magda-dev.terria.io" for `externalUrl`?
`Open in National Button` seems only work with `https` URL for cross-domain request

Reply:
Dev site should be ok - externalUrl is still present on line 4, I've just made correspondence api use that one.

Would probably be wise to override it for the ui-only previews though 🤔

Reply:
I think it's good enough then~ 👍

Reply:
Probably don't need to override it as I guess one day gitlab preview will support `https`